presidio-anonymizer/presidio_anonymizer/anonymizers/fpe.py

```diff
@@ -14,10 +14,14 @@ class FPE(Anonymizer):
     PII text will be replaced with a format preserving encryption using FF1 algorithm.
     """
 
-    def anonymize(self, original_text=None, params={}):
+    def anonymize(self, text: str = None, params: dict = None) -> str:
```

Comment:
Why do we need the default values?

Reply:
By itself, the anonymizer implementation might not need the original_text or parameters. 
If it is called not by the engine, for example. The called will have to know to call the validate as we discussed.